azure/template.json

```diff
@@ -35,6 +35,12 @@
         "description": "The custom hostname to add to the app service."
       }
     },
+    "customDomains": {
+      "type": "array",
+      "metadata": {
+        "description": "The custom hostname to add to the app service."
```

Comment:
Should it describe custom domains instead?

Reply:
Actually, `api` and `api2` are the hostname segment, no? Should this be `fqdn`?

Reply:
> 
> 
> Should it describe custom domains instead?

Descriptions now corrected......